src/renderers/testing/ReactTestTextComponent.js

```diff
@@ -15,6 +15,7 @@
 import type { ReactText } from 'ReactTypes';
 
 class ReactTestTextComponent {
+  _currentElement: ReactText;
```

Comment:
@zpao I added the second `class` usage in the codebase last week, so it's probably consistent so far :p